src/search.cpp

```diff
@@ -724,7 +723,6 @@ namespace {
         ss->staticEval = eval = VALUE_NONE;
         improving = false;
         improvement = 0;
-        complexity = 0;
```

Comment:
@pb00067 
Removing this line is dangerous. Currently complexity isn't used in the move loop but it can and then it introduce an undefined behavior. So for robustness always an initialization should be done.

Reply:
>>then it introduce an undefined behavior. 

Our valgrind framework will prevent that, so no need to take this precautions nowadays.

Reply:
Sorry, using such tools doesn't mean not to write robust code.

Reply:
In this specific case, if somebody tries to use _complexity_ unconsciously also when in check, he will have a problem anyway, since _complexity_ in most positions is quite above 0.
Most times when someone tries uninitialized stuff  on fishtest , the test aborts as soon a unix machine is joining the test (will produce a different bench).